python/prairielearn.py

```diff
@@ -75,6 +77,10 @@ def to_json(v):
     """
     if np.isscalar(v) and np.iscomplexobj(v):
```

Comment:
Let's add a quick variable name extractor: 

```suggestion
    # Retrieve the name of the `v` variable passed in by _inspecting_ the frame. 
    # Note, this approach does _not_ scale to multiple frame calls. Hence, we're opting not to functionalize it.
    variable_name = [k for k,x in inspect.currentframe().f_back.f_locals.items() if x is v][0]
    if np.isscalar(v) and np.iscomplexobj(v):
```